compiler/rustc_parse_format/src/lib.rs

```diff
@@ -385,11 +382,7 @@ impl<'a> Parser<'a> {
     /// Notifies of an error. The message doesn't actually need to be of type
     /// String, but I think it does when this eventually uses conditions so it
     /// might as well start using it now.
-    fn err_with_note<
-        S1: Into<string::String>,
-        S2: Into<string::String>,
-        S3: Into<string::String>,
-    >(
+    fn err_with_note<S1: Into<String>, S2: Into<String>, S3: Into<String>>(
```

Comment:
This is not because of you, it's just a fmt change, but nowadays we'd likely have written these using `impl Into<String>`.

Reply:
Fixed both here and just above for `fn err`.